packages/client/src/components/MarkdownParser.js

```diff
@@ -0,0 +1,12 @@
+import Body from "./styled/Body";
```

Comment:
I suppose I missed this, but we really shouldn't be calling things `Body` if it's not related to [`document.body`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/body)